### ISSUE TITLE
Update examples README

### ISSUE THREAD

**Issue by fredRos** [bat/bat#77]
* convert to markdown
* update directory names (camelCase), `expert` gone...

**Comment by fredRos** [comment]
This should go into the readme: If I enter a directory of an example, play around and rerun it, I get cling warnings
```
~/workspace/bat/examples/basic/rootOutput {master}$ ./readRootOutput 

RooFit v3.60 -- Developed by Wouter Verkerke and David Kirkby 
                Copyright (C) 2000-2013 NIKHEF, University of California & Stanford University
                All rights reserved, please read http://roofit.sourceforge.net/license.txt

Error in cling::AutoloadingVisitor::InsertIntoAutoloadingState:
   Missing FileEntry for BAT/BCCauchyPrior.h
   requested to autoload type BCCauchyPrior
Error in cling::AutoloadingVisitor::InsertIntoAutoloadingState:
   Missing FileEntry for BAT/BCConstantPrior.h
   requested to autoload type BCConstantPrior
...
```
unless I prepare the environment with
```sh
. ../../../test-env.sh
```

This is inconvenient!

**Comment by oschulz** [comment]
I'll look into that ...

**Comment by oschulz** [comment]
Now you broke it completely ;-)

    [...]
    make[2]: Entering directory `[...]/bat/examples'
    make[2]: *** No rule to make target `README', needed by `distdir'.  Stop.

I guess something is missing in dist.

**Comment by fredRos** [comment]
Good catch, buildserver! I just ran `make check` and not `make distcheck`. It is fixed now in the updated #98 .

I was aware of that dist problem and looked into `Makefile.am` but it didn't list that file, instead it was in `examples/Makefile.am`. This is slightly confusing as the main `Makefile.am` lists a number of such files in the subdirectories but apparently it doesn't include all subdirectories.

```

EXTRA_DIST= \
	INSTALL.md \
	README.md \
	readme.devel \
	models/base/LDef.h  \
	models/base/LDef_roostats.h \
	doc/COPYING doc/CREDITS doc/LICENSE doc/ChangeLog doc/releasenotes.md \
	doc/introduction/bat.pdf  doc/introduction/introduction.tex  doc/introduction/Makefile \
	doc/quickstart/bat.pdf  doc/quickstart/Makefile  doc/quickstart/quickstart.tex \
	doc/customdoxygen.css  doc/Doxyfile \
	examples \
	tools/bat-project.in
```
I had three build directories for three version of root. It turns out `make distcheck` doesn't work from an out-of-source build because example files are missing. I will gladly turn this into another issue and assign it to you :smile:
